lib/transport/tls-context.h

```diff
@@ -54,6 +54,7 @@ typedef enum
   TSO_NOTLSv12=0x0010,
   TSO_NOTLSv13=0x0020,
   TSO_IGNORE_UNEXPECTED_EOF=0x0040,
+  TSO_NOHOSTNAMECHECK=0x0080,
```

Comment:
can you rename this so it's somewhat easier to read?

TSO_IGNORE_HOSTNAME_MISMATCH ?

Reply:
Done